mailpoet/assets/js/src/automation/integrations/mailpoet/analytics/components/tabs/automation_flow/index.tsx

```diff
@@ -10,6 +10,7 @@ Hooks.addFilter(
     function statisticSeperatorWrapper(previousStepData: StepData) {
       return <StatisticSeparator previousStepId={previousStepData.id} />;
     },
+  20,
```

Comment:
Just curious, why is this needed? Would it make sense to include a comment about this?

Reply:
Not sure if I understand what you mean. Do you mean `<StatisticSeparator />`? It is this element: 
![image](https://github.com/mailpoet/mailpoet/assets/6458412/eca47157-522c-4495-9f1d-a5920e03f067)

Or do you mean the callback? I think it is a callback so that we can apply the filter once and then produce the separator to connect/"separate" different steps with each other.

Do you have an idea, how to comment that meaningful?

Reply:
Sorry I wasn't clearer! I was specifically wondering about the priority of the callback. Looking at it again I see why you would want this to come after other hooks have had a chance to fire since you wouldn't want the view version to get replaced by a different callback. Sometimes it can become tricky keeping track of callback priorities, but in this case it's actually pretty straightforward (unless I'm actually misinterpreting it!).

I don't think there's a need to comment this on second glance.